### ISSUE TITLE
Like and dislike buttons.

### ISSUE THREAD

**Issue by Niklan** [GPMDP/google-play-music-desktop-remote#10]
Hi. Android app is great and helpful sometimes, but problem is i can't like the track currently playing, so i need to go to PC, minimize all apps (because hotkeys is not working in fullscreen apps), and then like the track.

**Comment by KruyKaze** [comment]
Same here. Thanks

**Comment by arjvik** [comment]
Yes, I was looking for this feature too. Do you know if this is available in the JSON/WebSocket API, so that I can make my own app/workflow to do this? Or is this an enhancement on the GPMDP side?

**Comment by xRADxRYANx** [comment]
+1 for this request! :)
